.github/classroom/autograding.json

```diff
@@ -0,0 +1,14 @@
+{
+  "tests": [
+    {
+      "name": "Test acceptance criteria",
+      "setup": "yarn install",
```

Comment:
Worth adding the `--frozen-lockfile` flag to ensure reproducible dependencies. Like you would with a CI

Reply:
Good shout, will do. 👍🏾